td-uefi-pi/fuzz/fuzz_targets/fuzzlib.rs

```diff
@@ -26,15 +27,28 @@ const CFV_FFS_HEADER_TRUST_ANCHOR_GUID: Guid = Guid::from_fields(
 ); // {77A2742E-9340-4AC9-8F85-B7B978580021}
 
 pub fn fuzz_hob_parser(buffer: &[u8]) {
-    if hob::get_hob_total_size(buffer).is_some() {
-        hob::dump_hob(buffer);
-        hob::get_system_memory_size_below_4gb(buffer);
-        hob::get_total_memory_top(buffer);
-        hob::get_fv(buffer);
-        hob::get_next_extension_guid_hob(buffer, &HOB_ACPI_TABLE_GUID);
-        hob::get_next_extension_guid_hob(buffer, &HOB_KERNEL_INFO_GUID);
-        hob::get_guid_data(buffer);
-        hob::seek_to_next_hob(buffer);
+    let mut test_buffer = buffer.to_vec();
+
+    // Update ptr in buffer
+    let ptr = test_buffer.as_ptr() as u64;
+    if test_buffer.len() >= 56 {
+        test_buffer[48..56].copy_from_slice(&u64::to_le_bytes(ptr + buffer.len() as u64)[..]);
```

Comment:
would you please add comment what is 48 and what is 56? Or use MACRO directly?

Reply:
Fixed